app/routes/schema.py

```diff
@@ -1,19 +1,20 @@
-from flask import Blueprint, jsonify
+from flask import Blueprint, Response, jsonify
 
 from app.utilities.schema import get_schema_list, load_schema_from_name
 
 schema_blueprint = Blueprint("schema", __name__)
 
 
 @schema_blueprint.route("/schemas/<schema_name>", methods=["GET"])
-def get_schema_json_from_name(schema_name):
+def get_schema_json_from_name(schema_name: str) -> Response | tuple[str, int]:
     try:
-        schema = load_schema_from_name(schema_name)
+        # Type ignore: load_schema_from_name will default the language_code param to None despite not being defaulted in method signature
+        schema = load_schema_from_name(schema_name)  # type: ignore
```

Comment:
This should work without type ignore?

Reply:
Yep seems you're right, looks like a remnant of when I was playing around with the language code type hinting. Sorted 👍